README.md

```diff
@@ -1,6 +1,8 @@
 # iop-gateway
 
-A lightweight and configurable gateway container designed to act as an entry point for deployments. It supports custom configuration mounting and includes identity handling behavior.
+A gateway gateway to and for Insights services.
+It acts as an entry point to Insights services and a gateway to Foreman for Insights services.
+It includes identity handling behavior.
```

Comment:
First sentence repeats gateway, I also wonder whether it make sense to avoid using Insights 🤷

Reply:
corrected